vmc/resource_vmc_cluster.go

```diff
@@ -165,7 +165,18 @@ func resourceClusterRead(d *schema.ResourceData, m interface{}) error {
 		d.SetId("")
 		return nil
 	}
-
+	clusterExists := false
+	for i := 0; i < len(sddc.ResourceConfig.Clusters); i++ {
```

Comment:
why not use range instead of a for loop?

Reply:
Done

Reply:
Done